.changeset/gold-cloths-fold.md

```diff
@@ -0,0 +1,6 @@
+---
+"@digdir/designsystemet": patch
+"@digdir/designsystemet-react": patch
+---
+
+Update dependencies
```

Comment:
To conform with our previous release notes we should write out which dependencies was updated to what version.
<img width="522" alt="image" src="https://github.com/user-attachments/assets/e652292c-8a43-47a4-aded-13c9cc2f9918" />